CondFormats/PCLConfig/plugins/AlignPCLThresholdsWriter.cc

```diff
@@ -65,10 +65,10 @@ AlignPCLThresholdsWriter::AlignPCLThresholdsWriter(const edm::ParameterSet& iCon
       m_minNrecords(iConfig.getParameter<unsigned int>("minNRecords")),
       m_parameters(iConfig.getParameter<std::vector<edm::ParameterSet> >("thresholds")) {
   //now do what ever initialization is needed
-  myThresholds = new AlignPCLThresholds();
+  AlignPCLThresholds myThresholds{};
 }
 
-AlignPCLThresholdsWriter::~AlignPCLThresholdsWriter() { delete myThresholds; }
+AlignPCLThresholdsWriter::~AlignPCLThresholdsWriter() {}
```

Comment:
Since now this is not doing anything you can use the default destructor

Reply:
yes you are correct! thanks for spotting this!

Reply:
done in 41e9574